test/abstract_syntax_tree.coffee

```diff
@@ -561,90 +561,121 @@ test "AST as expected for Index node", ->
     optional: no
     shorthand: no
 
-# test "AST as expected for Range node", ->
-#   testExpression '[x..y]',
-#     type: 'Range'
-#     exclusive: no
-#     equals: '='
-#     from:
-#       value: 'x'
-#     to:
-#       value: 'y'
-
-#   testExpression '[4...2]',
-#     type: 'Range'
-#     exclusive: yes
-#     equals: ''
-#     from:
-#       value: '4'
-#     to:
-#       value: '2'
-
-#   testExpression 'for x in [42...43] then',
-#     range: yes
-#     source:
-#       type: 'Range'
-#       exclusive: yes
-#       equals: ''
-#       from:
-#         value: '42'
-#       to:
-#         value: '43'
-
-#   testExpression 'for x in [y..z] then',
-#     range: yes
-#     source:
-#       type: 'Range'
-#       exclusive: no
-#       equals: '='
-#       from:
-#         value: 'y'
-#       to:
-#         value: 'z'
-
-#   testExpression 'x[..y]',
-#     properties: [
-#       range:
-#         type: 'Range'
-#         exclusive: no
-#         equals: '='
-#         from: undefined
-#         to:
-#           value: 'y'
-#     ]
+test "AST as expected for Range node", ->
+  testExpression '[x..y]',
+    type: 'Range'
+    exclusive: no
+    from:
+      name: 'x'
+    to:
+      name: 'y'
 
-#   testExpression 'x[y...]',
-#     properties: [
-#       range:
-#         type: 'Range'
-#         exclusive: yes
-#         equals: ''
-#         from:
-#           value: 'y'
-#         to: undefined
-#     ]
+  testExpression '[4...2]',
+    type: 'Range'
+    exclusive: yes
+    from:
+      value: 4
+    to:
+      value: 2
 
-#   testExpression 'x[...]',
-#     properties: [
-#       range:
-#         type: 'Range'
-#         exclusive: yes
-#         equals: ''
-#         from: undefined
-#         to: undefined
-#     ]
+  # testExpression 'for x in [42...43] then',
```

Comment:
I’m assuming these tests are commented out because we don’t yet support AST output for `for` or `in`?

Reply:
Yup exactly so just included tests for "standalone" ranges and slices for now